examples/nlp/sentence_classification_with_bert.py

```diff
@@ -166,3 +173,45 @@ def create_pipeline(num_samples=-1,
          optimization_params={"num_epochs": args.num_epochs,
                               "lr": args.lr,
                               "weight_decay": args.weight_decay})
+
+                              
```

Comment:
this example is nice but it shouldn't be here. i think we already have several examples of processors for various datasets data/datasets/utils.py. if you want to keep this example, we should also move it there.